pkg/services/k8s/apiserver/service.go

```diff
@@ -52,9 +54,22 @@ func (s *service) GetRestConfig() *rest.Config {
 
 func (s *service) start(ctx context.Context) error {
 	serverRunOptions := options.NewServerRunOptions()
+	serverRunOptions.Admission.GenericAdmission.EnablePlugins = []string{
+		"MutatingAdmissionWebhook",
```

Comment:
is this needed? i'm seeing the CRDs without it:

```
bin/grafana kubectl api-resources | rg Webhook
mutatingwebhookconfigurations                  admissionregistration.k8s.io/v1               false        MutatingWebhookConfiguration
validatingwebhookconfigurations                admissionregistration.k8s.io/v1               false        ValidatingWebhookConfiguration
```

Reply:
i dug into this a bit more, and it looks like they are enabled by default in [1.26.2](https://github.com/kubernetes/kubernetes/blob/fc04e732bb3e7198d2fa44efa5457c7c6f8c0f5b/pkg/kubeapiserver/options/plugins.go#L152-L153)